src/ui/mod.rs

```diff
@@ -1237,9 +1238,10 @@ where
           }
         }
         album_artist.push_str(&format!(
-          "{} - {}",
+          "{} - {} ({})",
           item.name.to_owned(),
-          create_artist_string(&item.artists)
+          create_artist_string(&item.artists),
+          item.album_type.as_deref().unwrap()
```

Comment:
Same here. Prefer `unwrap_or`

Reply:
Same fix in commit: 75f107f. Marking as Resolved.